isso/css/isso.css

```diff
@@ -92,6 +96,9 @@
     font-weight: bold;
     color: #555;
 }
+.isso-comment > .isso-text-wrapper > .isso-comment-header .isso-page-author {
+    color: #19798d !important;
```

Comment:
Please don't use `!important`, either make the selector more specific or fix the underlying issue.